src/Themes/Default/MultiSelectPromptRenderer.php

```
@@ -53,6 +62,21 @@ public function __invoke(MultiSelectPrompt $prompt): string
         };
     }
 
+    /**
+     * Calculate the description width based on options.
+     */
+    protected function calculateDescriptionWidth(Prompt $prompt, int $maxWidth): int
```

PHPMD (2.15.0)

Avoid unused parameters such as '$maxWidth'. (Unused Code Rules)

(UnusedFormalParameter)